project/profiles/capacity4more/modules/c4m/og/c4m_og/c4m_og.module

```diff
@@ -2563,29 +2568,28 @@ function c4m_og_edit_author_autocomplete($string = '') {
     return;
   }
 
-  $gid = c4m_og_current_group_id();
-  // Return early if there is no group context.
+  // Resolve group id from auto complete path, which is set to
+  // 'edit-author/autocomplete/<$gid>'.
+  $destination = drupal_get_destination();
```

Comment:
Tried to do this, but autocomplete is not as other menu items.
The parameter it gets is string to autocomplete.
`  $items['edit-author/autocomplete/%node'] = array(
    'page callback' => 'c4m_og_edit_author_autocomplete',`

`function c4m_og_edit_author_autocomplete($string = '')`